src/bokeh/colors/color.py

```diff
@@ -207,7 +208,12 @@ class RGB(Color):
 
     '''
 
-    def __init__(self, r: int, g: int, b: int, a: float = 1.0) -> None:
+    r: int
+    g: int
+    b: int
+    a: float
+
+    def __init__(self, r: int | np.uint8, g: int | np.uint8, b: int | np.uint8, a: float | np.floating = 1.0) -> None:
```

Comment:
```suggestion
    def __init__(self, r: int | np.integer, g: int | np.integer, b: int | np.integer, a: float | np.floating = 1.0) -> None:
```

I don't think it should be limited to `np.uint8` even though that is the correct range for RGB.